config/development.toml

```diff
@@ -731,5 +731,11 @@ encryption_key = "A8EF32E029BC3342E54BF2E172A4D7AA43E8EF9D2C3A624A9F04E2EF79DC69
 [locker_based_open_banking_connectors]
 connector_list = ""
 
+[grpc_client]
```

Comment:
This might note be rquired

Reply:
This is so we can have all the hosts and ports for all the clients under one `grpc_client` struct